llama_stack/distribution/datatypes.py

```diff
@@ -225,6 +225,7 @@ class Provider(BaseModel):
     provider_id: str
     provider_type: str
     config: Dict[str, Any]
+    health: Optional[str] = None
```

Comment:
why is this needed here -- this object representes information obtained from run.yaml where health should not be relevant? (I know we have too many confusing "Provider" classes around)